scripts/build-dependencies.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+set -e
+
+function findNode() {
+    return $(which npm 1>/dev/null 2>&1)
+}
+
+while [[ $# -gt 0 ]]
+do
+key="$1"
+
+case $key in
+    --no-color)
+    NOCOLOR=true
+    ;;
+    *) # If it wasn't --no-color, we assume it was the path of the root of the project
+    SUPPLIED_APP_FOLDER_PATH=$key
+    ;;
+esac
+shift
+done
+
+MYPATH=$( cd $SUPPLIED_APP_FOLDER_PATH && pwd )
```

Comment:
Should we check that `$SUPPLIED_APP_FOLDER_PATH` actually exists?  Or are we just depending on bash to error if its an invalid path?